### PR TITLE
DCP-3398 : Remvoing distribution/README.txt and updating gradle to put correct ReadMe in zip

### PR DESCRIPTION
**Pull request by swatisuman1990**

### CODE REVIEW

**Review by suranjayathilaka — CHANGES_REQUESTED** [review]
make sure everything that was in the removed readme is not missing in the top level readme

**Comment by swatisuman1990** [comment]
> make sure everything that was in the removed readme is not missing in the top level readme

The ReadMe in distribution was an old version of the Project level ReadMe. The current one is just an updated version.

File: distribution/README.txt [hunk 1]
```diff
@@ -1,68 +0,0 @@
...
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- ******************************************************************************/
-
-version:unspecified
-Ena File Downloader
-Copyright © EMBL 2021 | EMBL-EBI is part of the European Molecular Biology Laboratory
-
-For support/issues, please contact us at https://www.ebi.ac.uk/ena/browser/support
```
Comment (suranjayathilaka) [inline distribution/README.txt:21]: what about all this ?

File: distribution/README.txt [hunk 2]
```diff
@@ -1,68 +0,0 @@
...
-java -jar ena-file-downloader.jar --accessions=SAMEA3231268,SAMEA3231287 --format= READS_FASTQ --location=C:\location --protocol=FTP --asperaLocation=null --email=youremail@email.com
-
-* `--accessions` comma separated list of accessions or the file path with accessions that one wishes to download.
-* `--format` The format for the download (`eg : READS_FASTQ,READS_SUBMITTED,ANALYSIS_SUBMITTED,ANALYSIS_GENERATED`)
-* `--location` The location for the download
-* `--protocol` The protocol to be used for download.(`eg : FTP, ASPERA`)
-* `--asperaLocation` The location of local Aspera Connect/CLI folder. Required if protocol is Aspera.
-* `--email` (Optional) The email at which one wishes to receive an alert on completion.
-
-Privacy Notice
```
Comment (suranjayathilaka) [inline distribution/README.txt:61]: and this?

**Review by suranjayathilaka — APPROVED** [review]

**Pull request merged**